src/meadowflow/event_log.py

```diff
@@ -105,9 +120,7 @@ def append_event(self, topic_name: TopicName, payload: T) -> None:
         # that the call_subscribers_loop will happen once per event--the loop might get
         # "backed up" until multiple events have been posted first, and then multiple
         # events will get processed in the same iteration of the loop.
-        self._event_loop.call_soon_threadsafe(
-            lambda: self._notify_call_subscribers.set()
-        )
+        asyncio.get_running_loop().call_soon(self._notify_call_subscribers.set)
```

Comment:
I don't think there's any reason to do this instead of just calling `self._notify_call_subscribers.set()` directly here? I only had the `call_soon_threadsafe` thing in here because I was assuming `append_event` could be called from a different asyncio event loop than "the" asyncio event loop that `call_subscribers_loop` runs on. But I think with the changes you've made that is no longer possible--there's only one asyncio event loop in the whole process?